src/modules/dashboard/components/TaskList/TaskListItem.tsx

```diff
@@ -50,42 +44,24 @@ interface InnerProps extends Props, EnhancerProps {}
 const displayName = 'dashboard.TaskList.TaskListItem';
 
 const TaskListItem = ({
+  colonyAddress,
+  colonyName,
   data,
   intl: { formatMessage },
   history,
 }: InnerProps) => {
-  const {
-    data: task,
-    entry: [colonyAddress, draftId],
-    isFetching: isFetchingTask,
-  } = data;
   const defaultTitle = formatMessage(MSG.untitled);
-  const {
-    workerAddress = undefined,
-    payouts = [],
-    reputation = undefined,
-    title = defaultTitle,
-  } = task || {};
-
-  const { data: colonyName, isFetching: isFetchingColonyName } = useDataFetcher(
-    colonyNameFetcher,
-    [colonyAddress],
-    [colonyAddress],
-  );
+  const { id: draftId, assignedWorker, title = defaultTitle } = data;
+
+  // @todo get payouts from centralized store
```

Comment:
Yeah, I think payouts are definitely more pressing than reputation so this should probably be a fixme (also the other occurrences)